docs/products/clickhouse/howto/query-stats.rst

```diff
@@ -0,0 +1,22 @@
+Query statistics
+=================
+Usually, Query statistics in ClickHouse can be obtained using the ``system.query_log`` table, which stores statistics of each executed query, including memory usage and duration. Aiven for ClickHouse® currently do not give access to the table ``system.query_log`` table, in order to directly query the table for the required statistics.
+
+Instead,use one of the two approach to view *Query Statistics* for Aiven for ClickHouse®
```

Comment:
```suggestion
Instead, use one of the two approaches to view *Query Statistics* for Aiven for ClickHouse®.
```

Reply:
Please rephrase into:

To fetch query statistics in Aiven for ClickHouse, you can use either Aiven Console or Aiven API.